src/kOS.Safe/Compilation/Opcode.cs

```diff
@@ -744,7 +744,7 @@ public class OpcodeSetMember : Opcode
         public override void Execute(ICpu cpu)
         {
             Structure value = cpu.PopStructureEncapsulated();         // new value to set it to
-            string suffixName = cpu.PopStack().ToString().ToUpper();  // name of suffix being set
+            string suffixName = cpu.PopStack().ToString();            // name of suffix being set
```

Comment:
I've walked the code searching every override for `GetSuffix` and `SetSuffix` and all have already been converted to case insensitive checks.  We should have no issues with replacing the logic.